testsuite/integration/basic/src/test/config/arq/arquillian.xml

```diff
@@ -7,7 +7,7 @@
     <container qualifier="jboss" default="true">
         <configuration>
             <property name="jbossHome">${basedir}/target/wildfly</property>
-            <property name="javaVmArguments">${server.jvm.args}</property>
+            <property name="javaVmArguments">${server.jvm.args} -agentlib:jdwp=transport=dt_socket,address=8787,server=y,suspend=n</property>
```

Comment:
This looks like development cruft. We use maven property surefire.jpda.args to control this kind of thing. That's already pretty complex so we shouldn't add other one-off things.

Reply:
Oops, sorry! Will send a PR right after this gets merged.